config/rewrites.d/developer.rackspace.com.json

```diff
@@ -230,13 +230,29 @@
             "status": 301
         },
         {
+           "description:": "Redirect metrics links with old product name",
             "from": "^\\/docs\\/cloud-metrics\\/v1\\/developer-guide\\/?(.*)$",
             "to": "/docs/metrics/v1/",
             "rewrite": false,
             "status": 301
         },
         {
-            "from": "^\\/docs\\/cloud-metrics\\/v2\\/developer-guide\\/?(.*)$",
```

Comment:
so the monitoring redirects had some copy-paste errors.  I updated them to point to correct product URL and version. (They were pointing to metrics.)